include/loadfragment2.h

```diff
@@ -0,0 +1,29 @@
+#ifndef LOADFRAGMENT2_H
```

Comment:
Oh, I like this header. We could consider adding it for MM and OoT too :eyes:

Reply:
I've added it to my load PR in MM as `z64load.h`. As for OOT I'm not sure, I think they are waiting to split headers?